packages/vats/src/core/chain-behaviors.js

```diff
@@ -16,6 +16,7 @@ import {
 import { importBundle } from '@endo/import-bundle';
 
 import * as Collect from '@agoric/run-protocol/src/collect.js';
+import * as STORAGE_PATH from '@agoric/cosmic-swingset/src/chain-storage-paths.js';
```

Comment:
hm, if packages/cosmic-swingset has a `dependencies` on packages/vats , `chain-storage-paths.js` might need to move into `packages/vats` to avoid a circular dependency